service/controllers/messages.go

```diff
@@ -624,3 +684,101 @@ func processImageResponses(result string) []messageQueryBody {
 	return queryResultArray
 
 }
+
+func processMeaning(response string) []meaningStr {
+
+	fmt.Println("this is the response")
+
+	found := false
+	
+	var subs1 = "meaning"
+	var subs2 = "example"
+	var subs3 = "subMeaning"
+	var subs4 = "subExample"
+	var subsLen1 = len(subs1)
+	var subsLen2 = len(subs2)
+	var subsLen3 = len(subs3)
+	var subsLen4 = len(subs4)
+	var mid = 0
+	var last = 0
+
+	var meaningBody meaningStr
+	var meaningBodyArray []meaningStr
+	var subMeaningBody submeanStr
+	var subMeaningBodyArray []submeanStr
+
+	for i:=0; i< len(response) - 200; i++ {
```

Comment:
please justify the hard-coded **200**

Reply:
It was because of an error "Slice bounds out of range", i have fixed it now properly.